brave-snouts-frontend/src/app/features/auction-feature/auction/auction-form/auction-form.component.ts

```diff
@@ -94,6 +94,8 @@ export class AuctionFormComponent implements OnInit {
       name: [auction.name, [Validators.required, Validators.minLength(1), Validators.maxLength(100)]],
       startDate: [auction.startDate, [Validators.required]],
       endDate: [auction.endDate, [Validators.required]],
+      startTime: [auction.startTime, [Validators.required]],
+      endTime: [auction.endTime, [Validators.required]]
```

Comment:
Parse out start and end time as local variables in this function and give them to FormControl as value

let startTime = ..get start time here from startDate..

startTime: [startTime, Validators.required]